src/BleScanner.cpp

```diff
@@ -0,0 +1,62 @@
+#include "BleScanner.h"
+#include <NimBLEUtils.h>
+#include <NimBLEScan.h>
+#include <NimBLEAdvertisedDevice.h>
+
+BleScanner::BleScanner(int reservedSubscribers)
```

Comment:
Please use the same coding style as we do, i.e. space after `if` and opening `{` on same line
We're using ArtisticStyle to reformat, based on .astylerc file in root of repository, but feel free to use your own styling tool